e2e/test/scenarios/embedding/sdk-iframe-embedding/custom-elements-api.cy.spec.ts

```diff
@@ -389,6 +389,46 @@ describe("scenarios > embedding > sdk iframe embedding > custom elements api", (
     });
   });
 
+  describe("<metabase-metabot>", () => {
+    it("should load the embedded Metabot component", () => {
+      H.visitCustomHtmlPage(`
+      ${H.getNewEmbedScriptTag()}
+      ${H.getNewEmbedConfigurationScript()}
+      <metabase-metabot />
+      `);
+
+      H.getSimpleEmbedIframeContent().within(() => {
+        cy.log("metabot chat should be interactive");
+        cy.findByText("Ask questions to AI.").should("be.visible");
+        cy.findByPlaceholderText("Ask AI a question...").type("Foo{enter}");
+        cy.findByText(
+          "Metabot is currently offline. Please try again later.",
```

Comment:
We already have plenty of Metabot tests on the SDK side (metabot-question.cy.spec.ts), so we only need a smoke test for EAJS.